.gitignore

```diff
@@ -25,6 +25,7 @@ package-lock.json
 # local environment
 .environment.yaml
 .user.yaml
+.org_agent_user.yaml
```

Comment:
BTW this shouldn't be needed as long as your file is in the root as we git ignore all YAML files that start with a dot:

```
/.*.yaml
```

Reply:
I will remove it thanks